scalding-hadoop-test/src/test/scala/com/twitter/scalding/platform/PlatformTest.scala

```diff
@@ -702,4 +723,21 @@ class PlatformTest extends WordSpec with Matchers with HadoopSharedPlatformTest
     }
   }
 
+  "An InvalidSourceTap that gets past validation" should {
+    "throw an InvalidSourceException" in {
+      val result = Try {
+        HadoopPlatformJobTest(new ReadPathJob(_), cluster)
+          .arg("input", "/sploop/boop/doopity/doo/")
+          .run
+      }
+
+      result match {
+        case Failure(fe: FlowException) if Option(fe.getCause).exists(_.isInstanceOf[InvalidSourceException]) => ()
```

Comment:
can we have an assert in here? otherwise scalatest won't count it in the test count.